config/environments/production.rb

```
@@ -100,6 +100,9 @@
   config.action_mailer.default_url_options = { host: ENV["ROOT_URL"] }
   config.action_mailer.delivery_method = :smtp
 
+  config.action_controller.asset_host = "#{ENV["ASSET_HOST"]}"
```

Please use fallback solution - or throw errors when these variables are not set. Otherwise, this can lead to weird bugs, hard to hunt.